documentation/3.4.0/setup/native/with-survey/ubuntu/README.md

```diff
@@ -0,0 +1,291 @@
+<div align="center">
```

Comment:
@MallanagoudaB read through the readme , make sure survey service aspects are also mentioned in the document